index.html

```diff
@@ -83,10 +83,17 @@ <h3 class="menu-label">menu</h3>
         <div class="locText end">&nbsp;</div><span class="filter-label">end</span>
     </div>
 
+    <div id="error-box" class="modal">
+        <h3 class="modal-header" role="alert">Error</h3>
+        <span id="error-text">moo!</span>
+        <button>Close</button>
+    </div>
+
     <!-- ======= map ======= -->
     <div id="map"></div>
 
     <script src="https://code.jquery.com/jquery-1.12.4.js" integrity="sha256-Qw82+bXyGq6MydymqBxNPYTaUXXq7c8v3CwiYwLLNXU=" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js"></script>
```

Comment:
Can we use SRI here too?

```
<script src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js" integrity="sha256-eGE6blurk5sHj+rmkfsGYeKyZx3M4bG+ZlFyA7Kns7E=" crossorigin="anonymous"></script>
```

![screenshot 2016-08-19 16 36 41](https://cloud.githubusercontent.com/assets/10409657/17823554/11015934-662b-11e6-99e0-0b703aef4a82.png)